apps/web-giddh/src/app/shared/header/components/new-manage-groups-accounts/manage-groups-accounts.component.ts

```diff
@@ -231,11 +237,13 @@ export class ManageGroupsAccountsComponent implements OnInit, OnDestroy, AfterVi
         let res: any[] = response;
         this.customFieldForm = this.createCustomFieldForm();
         const customRow = this.customFieldForm.get('customField') as FormArray;
-        res.map(item => {
-            item.isEditMode = true;
-            customRow.push(this.initNewCustomField(item));
-        });
-        this.removeCustomFieldRow(0);
+        if (res.length) {
```

Comment:
Do add nullity check on `res`

Reply:
No need to add nullity check it local var array